src/EFCore/Metadata/Builders/IConventionAnnotatableBuilder.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.EntityFrameworkCore.Metadata.Builders
     ///         not used in application code.
     ///     </para>
     /// </summary>
+    /// <remarks>
+    ///     See <see href="https://aka.ms/efcore-docs-conventions">EF Core model building conventions</see> for more information.
```

Comment:
We don't seem to have "EF Core" in the other links (and that seems contextually ok)

```suggestion
    ///     See <see href="https://aka.ms/efcore-docs-conventions">EF Core model building conventions</see> for more information.
```
```suggestion
    ///     See <see href="https://aka.ms/efcore-docs-conventions">Model building conventions</see> for more information.
```